src/command/output.rs

```diff
@@ -460,13 +457,13 @@ impl RoverOutput {
 
 fn print_descriptor(descriptor: impl Display) -> io::Result<()> {
```

Comment:
What does "descriptor" mean in this context? I couldn't figure it out, but the code seems to print a line with header-like formatting.

Reply:
Totally - we print a `descriptor` to `stderr` for each command output. So like for `rover subgraph fetch` we let folks know that the output they are looking at is `SDL`.